frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentStepper.tsx

```diff
@@ -55,17 +55,23 @@ export const PublicEnrollmentStepper = ({
   const isStepCompleted = (stepNumber: number) =>
     stepNumber < activeStep && !(isError && stepNumber === doneStepNumber);
 
-  const mobileStepValue = activeStep * (100 / doneStepNumber);
-  const mobilePhaseText = `${activeStep}/${doneStepNumber}`;
+  const stepValue =
+    activeStep >= PublicEnrollmentFormStep.Done
+      ? PublicEnrollmentFormStep.Done
```

Comment:
Vähän tätä ratkaisua pohdiskelin ja mietin olisko tää `stepValue` nyt oikea tieto tuossa `mobileAriaLabel`:in sisällön osalta, mutta pitäiskö mobileStepValuen sekä mobilePhaseTextin osalta `PublicEnrollmentFormStep.Done` käyttää `doneStepNumber`:ia ?